From 8f4722f392622d5ce50934d77c97ea8f94146e68 Mon Sep 17 00:00:00 2001
From: maximilian attems <max@stro.at>
Date: Sun, 4 May 2008 23:19:29 +0200
Subject: [PATCH] [USB] c67x00 build fix

drivers/usb/c67x00/c67x00-ll-hpi.c: In function `ll_recv_msg':
drivers/usb/c67x00/c67x00-ll-hpi.c:243: error: `HZ' undeclared (first use in this function)

Signed-off-by: maximilian attems <max@stro.at>
---
 drivers/usb/c67x00/c67x00-ll-hpi.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/c67x00/c67x00-ll-hpi.c b/drivers/usb/c67x00/c67x00-ll-hpi.c
index f3430b3..9e0430f 100644
--- a/drivers/usb/c67x00/c67x00-ll-hpi.c
+++ b/drivers/usb/c67x00/c67x00-ll-hpi.c
@@ -24,6 +24,7 @@
 #include <asm/byteorder.h>
 #include <linux/io.h>
 #include <linux/usb/c67x00.h>
+#include <linux/jiffies.h>
 #include "c67x00.h"
 
 #define COMM_REGS 14
-- 
1.5.5.1

